assets/css/_vehicle_map.scss

```diff
@@ -177,38 +177,6 @@
   border-top-color: $color-tooltip-background;
 }
 
-// Adding .leaflet-control-container increases specificity to make sure we override
-// base Leaflet control positioning
-.leaflet-control-container .m-vehicle-map__street-view-control {
```

Comment:
btw, [this commit `f303566` (#1921)](https://github.com/mbta/skate/pull/1921/commits/f3035669520d6300c328dd2c483fe27cf0111b64) allows us to drop the `.leaflet-control-container` to override specificity.